src/Tempest/Database/src/Migrations/MigrationManager.php

```diff
@@ -108,6 +110,60 @@ public function dropAll(): void
         }
     }
 
+    public function rehashAll(): void
+    {
+        try {
+            $existingMigrations = Migration::all();
+        } catch (PDOException) {
+            return;
+        }
+
+        foreach ($existingMigrations as $existingMigration) {
+            $databaseMigration = array_find(
```

Comment:
What exactly are we doing here? Shouldn't we just loop over the existing migrations and update them?

Reply:
Aaah that makes sense! Could you add that explanation as a comment in the code? Just so that we remember two months from now :)